src/components/BTextArea/BTextArea.test.js

```diff
@@ -0,0 +1,51 @@
+import { shallow } from 'vue-test-utils'
+import BTextarea from './BTextarea'
```

Comment:
`BTextArea`. It seems like it shouldn't work the way it's written because the casing is different. Weird.

Reply:
Huh? The casing is the same. The file name might be differently cased, though I thought I changed that.

Reply:
I see what's happening. On my local, the files are actually `Textarea` but apparently git doesn't care about that. I wonder if it would break if someone pulled it down.

Reply:
I see both `TextArea` and `Textarea` and not just for the filename but for the directory too. Should be consistent

Reply:
It runs fine locally, which is very strange because I still see the directory as `TextArea`